### PR TITLE
[SETUP][MMSYS][SHELL32] Adding missing French translation (and typo fix on existing)

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

- 2 typos in MMSYS French ressource file
- Setup ressource for French is missing
- 1 missing item in shell32 French ressource file

## Proposed changes

- Fix 2 typos in MMSYS French ressource file
- Complete Setup ressource for French
- Full shell32 French ressource file

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-french

**Comment by KyleKatarn31** [comment]
@HBelusca : Review is welcome :-)

File: base/setup/reactos/lang/fr-FR.rc [hunk 1]
```diff
@@ -53,76 +53,76 @@ CAPTION "Installation de ReactOS"
...
+    LTEXT "Ordinateur :", IDC_STATIC, 18, 41, 74, 11
     EDITTEXT IDC_COMPUTER, 95, 40, 210, 13, ES_READONLY | ES_AUTOHSCROLL | WS_VISIBLE | NOT WS_BORDER | NOT WS_TABSTOP
-    LTEXT "Display:", IDC_STATIC, 18, 53, 74, 11
+    LTEXT "Affichage :", IDC_STATIC, 18, 53, 74, 11
     EDITTEXT IDC_DISPLAY, 95, 52, 210, 13, ES_READONLY | ES_AUTOHSCROLL | WS_VISIBLE | NOT WS_BORDER | NOT WS_TABSTOP
-    LTEXT "Keyboard:", IDC_STATIC, 18, 65, 74, 11
+    LTEXT "Clavier :", IDC_STATIC, 18, 65, 74, 11
     EDITTEXT IDC_KEYBOARD, 95, 64, 210, 13, ES_READONLY | ES_AUTOHSCROLL | WS_VISIBLE | NOT WS_BORDER | NOT WS_TABSTOP
-    LTEXT "Destination volume:", IDC_STATIC, 18, 77, 74, 11
+    LTEXT "Volume de destination:", IDC_STATIC, 18, 77, 74, 11
```
Comment (HBelusca) [inline base/setup/reactos/lang/fr-FR.rc:111]:
```suggestion
    LTEXT "Volume de destination :", IDC_STATIC, 18, 77, 74, 11
```

File: base/setup/reactos/lang/fr-FR.rc [hunk 2]
```diff
@@ -132,44 +132,44 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
 CAPTION "Installation de ReactOS"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Completing the ReactOS Setup Wizard", IDC_FINISHTITLE, 115, 8, 195, 24
-    LTEXT "You have successfully completed the first stage of ReactOS Setup.", IDC_STATIC, 20, 50, 277, 10
-    LTEXT "When you click Finish, your computer will restart.", IDC_STATIC, 20, 80, 277, 10
+    LTEXT "Finalisation de l'assistant d'installation de ReactOS", IDC_FINISHTITLE, 115, 8, 195, 24
+    LTEXT "Vous avez complété avec succès la première étape d'installation de ReactOS.", IDC_STATIC, 20, 50, 277, 10
+    LTEXT "Si vous appuyez sur Terminé, votre ordinateur va redémarrer.", IDC_STATIC, 20, 80, 277, 10
```
Comment (HBelusca) [inline base/setup/reactos/lang/fr-FR.rc:137]:
```suggestion
    LTEXT "Si vous appuyez sur Terminer, votre ordinateur va redémarrer.", IDC_STATIC, 20, 80, 277, 10
```

File: base/setup/reactos/lang/fr-FR.rc [hunk 3]
```diff
@@ -132,44 +132,44 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
...
     CONTROL "", IDC_RESTART_PROGRESS, "msctls_progress32", PBS_SMOOTH | WS_CHILD | WS_VISIBLE | WS_BORDER, 20, 120, 277, 8
-    LTEXT "If there is a CD in a drive, remove it. Then, to restart your computer, click Finish.", IDC_STATIC, 10, 180, 297, 20
+    LTEXT "S'il y a un CD dans le lecteur, retirer le. Ensuite pour redémarrer votre ordinateur appuyez sur Terminer.", IDC_STATIC, 10, 180, 297, 20
```
Comment (HBelusca) [inline base/setup/reactos/lang/fr-FR.rc:139]:
```suggestion
    LTEXT "S'il y a un CD dans le lecteur, retirez le. Appuyez ensuite sur Terminer pour redémarrer votre ordinateur.", IDC_STATIC, 10, 180, 297, 20
```

File: base/setup/reactos/lang/fr-FR.rc [hunk 4]
```diff
@@ -132,44 +132,44 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
...
-    IDS_RESTARTSUBTITLE "The first stage of setup has been completed, restart to continue with second stage"
-    IDS_SUMMARYTITLE "Installation Summary"
-    IDS_SUMMARYSUBTITLE "List installation properties to check before apply to the installation device"
-    IDS_ABORTSETUP "ReactOS is not completely installed on your computer. If you quit Setup now, you will need to run Setup again to install ReactOS. Really quit?"
-    IDS_ABORTSETUP2 "Abort installation?"
-    IDS_NO_TXTSETUP_SIF "Unable to find 'txtsetup.sif'.\nSetup is unable to continue."
+    IDS_TYPETITLE "Bienvenue dans l'installation de ReactOS"
+    IDS_TYPESUBTITLE "Choisissez un type d'installation."
+    IDS_DEVICETITLE "Configurer les périphériques de base."
+    IDS_DEVICESUBTITLE "Configrer l'affichage et le clavier."
```
Comment (HBelusca) [inline base/setup/reactos/lang/fr-FR.rc:147]:
```suggestion
    IDS_DEVICESUBTITLE "Configurer l'affichage et le clavier."
```

File: base/setup/reactos/lang/fr-FR.rc [hunk 5]
```diff
@@ -132,44 +132,44 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
...
+    IDS_DRIVETITLE "Configurer la partition d'installation et le dossier système."
+    IDS_DRIVESUBTITLE "Préparer la partition d'installation, le dossier système et le chargeur de démarrage."
+    IDS_PROCESSTITLE "Configurer la partition, copier les fichiers et configurer le système."
+    IDS_PROCESSSUBTITLE "Créer et formatter la partition, copier les fichier, installer et configurer le chargeur de démarrage"
```
Comment (HBelusca) [inline base/setup/reactos/lang/fr-FR.rc:151]:
```suggestion
    IDS_PROCESSSUBTITLE "Créer et formatter la partition, copier les fichiers, installer et configurer le chargeur de démarrage."
```

File: base/setup/reactos/lang/fr-FR.rc [hunk 6]
```diff
@@ -132,44 +132,44 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
...
+    IDS_RESTARTTITLE "Première étape de configuration terminée"
+    IDS_RESTARTSUBTITLE "La première étape de l'installation est terminée. Veuillez redémarrer pour continuer avec la seconde étape"
```
Comment (HBelusca) [inline base/setup/reactos/lang/fr-FR.rc:153]:
```suggestion
    IDS_RESTARTSUBTITLE "La première étape de l'installation est terminée. Veuillez redémarrer pour continuer avec la seconde étape."
```

File: base/setup/reactos/lang/fr-FR.rc [hunk 7]
```diff
@@ -132,44 +132,44 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
...
 END
 
 STRINGTABLE
 BEGIN
-    IDS_INSTALLATION_NAME "Name"
-    IDS_INSTALLATION_PATH "Installation Path"
-    IDS_INSTALLATION_VENDOR "Vendor Name"
+    IDS_INSTALLATION_NAME "Nom"
+    IDS_INSTALLATION_PATH "Chemin d'installation"
+    IDS_INSTALLATION_VENDOR "Nom de fabriquant"
```
Comment (HBelusca) [inline base/setup/reactos/lang/fr-FR.rc:166]: "fabriquant": participe présent. "fabricant": le nom (ou adjectif).

Comment (HBelusca) [inline base/setup/reactos/lang/fr-FR.rc:166]:
```suggestion
    IDS_INSTALLATION_VENDOR "Nom du fabricant"
```

File: dll/cpl/mmsys/lang/fr-FR.rc [hunk 8]
```diff
@@ -27,12 +27,12 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
...
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Un schéma sons est un ensemble de sons appliqués aux évènements de ReactOS et aux programmes. Vous pouvez sélectionner un schéma existant ou en enregistrer que vous avez modifié.", -1, 8, 7, 230, 40
+    LTEXT "Un schéma sons est un ensemble de sons appliqués aux évènements de ReactOS et aux programmes. Vous pouvez sélectionner un schéma existant ou en enregistrer un que vous avez modifié.", -1, 8, 7, 230, 40
     LTEXT "S&chéma sons :", -1, 8, 42, 150, 17
     COMBOBOX IDC_SOUND_SCHEME, 8, 53, 230, 46, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     PUSHBUTTON "&Enregistrer sous...", IDC_SAVEAS_BTN, 120, 70, 65, 15
     PUSHBUTTON "Supprimer", IDC_DELETE_BTN, 188, 70, 50, 15
-    LTEXT "Pour changer les sons, cliquez sur un évènement de programme dans la liste suivante, puis sélectionnez un son à appliquer. Vous pouvez enregistrer les changements en tant que nouveau schéma son.", -1, 8, 90, 230, 40
+    LTEXT "Pour changer les sons, cliquez sur un évènement de programme dans la liste suivante, puis sélectionnez un son à appliquer. Vous pouvez enregistrer les changements en tant que nouveau schéma sons.", -1, 8, 90, 230, 40
```
Comment (HBelusca) [inline dll/cpl/mmsys/lang/fr-FR.rc:35]:
```suggestion
    LTEXT "Pour changer les sons, cliquez sur un événement de programme dans la liste suivante, puis sélectionnez un son à appliquer. Vous pouvez enregistrer les changements en tant que nouveau schéma sons.", -1, 8, 90, 230, 40
```

File: dll/cpl/mmsys/lang/fr-FR.rc [hunk 9]
```diff
@@ -27,12 +27,12 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Sons"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Un schéma sons est un ensemble de sons appliqués aux évènements de ReactOS et aux programmes. Vous pouvez sélectionner un schéma existant ou en enregistrer que vous avez modifié.", -1, 8, 7, 230, 40
+    LTEXT "Un schéma sons est un ensemble de sons appliqués aux évènements de ReactOS et aux programmes. Vous pouvez sélectionner un schéma existant ou en enregistrer un que vous avez modifié.", -1, 8, 7, 230, 40
```
Comment (HBelusca) [inline dll/cpl/mmsys/lang/fr-FR.rc:30]:
```suggestion
    LTEXT "Un schéma sons est un ensemble de sons appliqués aux événements de ReactOS et aux programmes. Vous pouvez sélectionner un schéma existant ou en enregistrer un que vous avez modifié.", -1, 8, 7, 230, 40
```

File: dll/cpl/mmsys/lang/fr-FR.rc [hunk 10]
```diff
@@ -27,12 +27,12 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
...
     LTEXT "S&chéma sons :", -1, 8, 42, 150, 17
     COMBOBOX IDC_SOUND_SCHEME, 8, 53, 230, 46, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     PUSHBUTTON "&Enregistrer sous...", IDC_SAVEAS_BTN, 120, 70, 65, 15
     PUSHBUTTON "Supprimer", IDC_DELETE_BTN, 188, 70, 50, 15
-    LTEXT "Pour changer les sons, cliquez sur un évènement de programme dans la liste suivante, puis sélectionnez un son à appliquer. Vous pouvez enregistrer les changements en tant que nouveau schéma son.", -1, 8, 90, 230, 40
+    LTEXT "Pour changer les sons, cliquez sur un évènement de programme dans la liste suivante, puis sélectionnez un son à appliquer. Vous pouvez enregistrer les changements en tant que nouveau schéma sons.", -1, 8, 90, 230, 40
     LTEXT "Évènements des programmes :", -1, 8, 118, 150, 17
```
Comment (HBelusca) [inline dll/cpl/mmsys/lang/fr-FR.rc:36]:
```suggestion
    LTEXT "Événements des programmes :", -1, 8, 118, 150, 17
```

File: base/setup/reactos/lang/fr-FR.rc [hunk 11]
```diff
@@ -132,44 +132,44 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
...
+    IDS_DEVICETITLE "Configurer les périphériques de base."
+    IDS_DEVICESUBTITLE "Configrer l'affichage et le clavier."
+    IDS_DRIVETITLE "Configurer la partition d'installation et le dossier système."
+    IDS_DRIVESUBTITLE "Préparer la partition d'installation, le dossier système et le chargeur de démarrage."
+    IDS_PROCESSTITLE "Configurer la partition, copier les fichiers et configurer le système."
+    IDS_PROCESSSUBTITLE "Créer et formatter la partition, copier les fichier, installer et configurer le chargeur de démarrage"
+    IDS_RESTARTTITLE "Première étape de configuration terminée"
+    IDS_RESTARTSUBTITLE "La première étape de l'installation est terminée. Veuillez redémarrer pour continuer avec la seconde étape"
+    IDS_SUMMARYTITLE "Résumé de l'installation"
+    IDS_SUMMARYSUBTITLE "Lister les propriétés d'installation pou vérifier avant d'appliquer sur le périphérique d'installation"
```
Comment (HBelusca) [inline base/setup/reactos/lang/fr-FR.rc:155]:
```suggestion
    IDS_SUMMARYSUBTITLE "Lister les propriétés d'installation à vérifier avant application sur le périphérique d'installation."
```

**Pull request merged**